src/coreclr/gc/CMakeLists.txt

```diff
@@ -104,6 +104,16 @@ list(APPEND GC_SOURCES ${GC_HEADERS})
 
 convert_to_absolute_path(GC_SOURCES ${GC_SOURCES})
 
+# clrgcr is build with standalone+regions
+if (CLR_CMAKE_TARGET_ARCH_ARM64 OR CLR_CMAKE_TARGET_ARCH_AMD64)
```

Comment:
this builds clrgcr.dll (and libclrgcr.so) on 64 bit platforms. We could change naming if required.

Reply:
NIT: for the sake of readability, would it be better to change the name to: clrgc_regions?

Reply:
didnt want to name it as regions since longer term it might be for a different feature. Perhaps something like experimental or something. Maybe rename it `clrgce` for experimental ?

Reply:
clrgcexp?

Reply:
yeah that works, will rename.